src/transformers/generation/flax_utils.py

```diff
@@ -0,0 +1,947 @@
+# coding=utf-8
```

Comment:
(No changes in this file, wasn't tagged as moved because of the new Mixin alias left in the original file. The same comment applies to the PT and TF changes)